docs/pages/playground.vue

```diff
@@ -8,22 +8,678 @@
+              </span>
+            </div>
+
+            <KGrid>
+              <KGridItem
+                :layout12="{ span: 8 }"
+                :layout8="{ span: 6 }"
+              >
+                <progress
+                  value="20"
+                  max="100"
+                  style="height:0.5em;width:100%"
+                ></progress>
+              </KGridItem>
+              <KGridItem
+                :layout12="{ span: 4 }"
+                :layout8="{ span: 2 }"
+              >
+                <div class="" style="float:right;">
+                  <KIcon icon="schedule" class="icon-size" />
+                  <KIcon icon="download" class="icon-size" />
+                </div>
+              </KGridItem>
+            </KGrid>
+          </div>
+        </template>
+      </KCard>
+    </ul>
+
+
+    <ul class="horizontal-cards-smaller">
+      <KCard
+        class="card"
+        :to="{ name: '/' }"
+        :headingLevel="2"
+        layout="horizontal"
+        thumbnailDisplay="none"
+      >
+        <template #title>
+          <KTextTruncator
+            text="This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title This is a sample resource title"
+            :maxLines="2"
+          />
+        </template>
+        <template #footer>
+          <div>
+            <div class="footer-top-header">
+              <span>
+                <KIcon icon="createShaded" class="icon-size" />
+                Practice
+              </span>
+              <span
+                :style="{ 'background-color': $themePalette.grey.v_100,'padding': '2px' }"
+              >
+                Short Activity
+              </span>
+            </div>
+
+            <KGrid>
+              <KGridItem
+                :layout12="{ span: 8 }"
+                :layout8="{ span: 6 }"
+              >
+                <progress
+                  value="20"
+                  max="100"
+                  style="height:0.5em;width:100%"
+                ></progress>
+              </KGridItem>
+              <KGridItem
+                :layout12="{ span: 4 }"
+                :layout8="{ span: 2 }"
+              >
+                <div class="" style="float:right;">
+                  <KIcon icon="schedule" class="icon-size" />
+                  <KIcon icon="download" class="icon-size" />
+                </div>
+              </KGridItem>
+            </KGrid>
+          </div>
+        </template>
+      </KCard>
+    </ul>
+
+    <ul class="horizontal-cards-smaller">
+      <KCard
+        class="card"
+        :to="{ name: '/' }"
+        :headingLevel="2"
+        layout="horizontal"
+        thumbnailDisplay="none"
+        thumbnailAlign="right"
+        title="This is a sample resource title"
```

Comment:
In this example I see there is more margin in the top than in the bottom, which causes the title to be slightly misaligned.

Reply:
I took care of this in https://github.com/learningequality/kolibri-design-system/pull/748